x-pack/plugins/observability_ai_assistant/public/components/chat/chat_body.tsx

```diff
@@ -26,11 +26,10 @@ import { MissingCredentialsCallout } from '../missing_credentials_callout';
 import { ChatHeader } from './chat_header';
 import { ChatPromptEditor } from './chat_prompt_editor';
 import { ChatTimeline } from './chat_timeline';
-import { KnowledgeBaseCallout } from './knowledge_base_callout';
 
 const containerClassName = css`
   max-height: 100%;
-  max-width: 100%;
+  max-width: 800px;
```

Comment:
The EuiInlineTextEdit component was proving to be a complete pain. Had to resort to this to make it not overflow the parent container.